stdlib/public/CMakeLists.txt

```diff
@@ -72,3 +72,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     add_subdirectory(SDK)
   endif()
 endif()
+
+if(WINDOWS IN_LIST SWIFT_SDKS)
+  add_subdirectory(SDK/Windows)
```

Comment:
I'm gonna be obstinate and say "SDK/" really means "Apple/" at this point. Maybe "WinSDK/" or "Windows/" instead of "SDK/Windows/"? cc @moiseev @lancep

Reply:
lol, thats why I wanted you to review this :)  This is getting into the territory of difficult things ... naming.  How about changing the structure to `SDK/Darwin` and `SDK/Windows`?  Or `Apple/` and `Microsoft/`?

Reply:
FWIW – The only other name for "Darwin" that covers all of Apple's operating systems is "xnu" (the kernel). It is near impossible to build an xnu-based platform without dragging along a ton of "Darwin" frameworks and daemons due to various reentrant dependancies.

Reply:
@davezarzycki - haha, perhaps we should call it "XNU/Darwin" (mirroring "GNU/Linux") :-p  I'm going to go with Darwin for now I think.